feather/base/src/anvil/player.rs

```diff
@@ -91,7 +91,7 @@ impl InventorySlot {
     }
 
     /// Converts an `ItemStack` and inventory position index into an `InventorySlot`.
```

Comment:
Technically you could link them with `[`ItemStack`]` (and the other one) but its not a requirement.